### PR TITLE
Blog for announcing OLS 1

### PR DESCRIPTION
**Pull request by malvikasharan**

### CODE REVIEW

**Comment by bebatut** [comment]
@malvikasharan here are the changes I was talking about yesterday.
I automatize the list of participants, mentors, projects and add links to the participants/project page.
Let me know if the changes are ok for you, otherwise I can revert them

**Comment by malvikasharan** [comment]
> @malvikasharan here are the changes I was talking about yesterday.
> I automatize the list of participants, mentors, projects and add links to the participants/project page.
> Let me know if the changes are ok for you, otherwise I can revert them

Ooh that sounds fancy! I am sure it's great but will leave you more  "oohs" after having a look.

File: _data/ols-1-projects.yaml [hunk 1]
```diff
@@ -144,8 +141,8 @@
     Bangladesh and Pakistan.<br>
     
     I want to initiate computational biology in Nepal to more number of enthusiasts
-    from different fields during this program. Getting the support from mentors, like
-    minded people and their mentoring will help for the sustainability of our project.
+    from different fields during this program. Getting the support from mentorss, like
```
[the remark this answers is not in the document]
Comment (bebatut) [reply _data/ols-1-projects.yaml]: oops...

[the remark this answers is not in the document]
Comment (bebatut) [reply _data/ols-1-projects.yaml]: it should be fixed

**Pull request merged**